pkg/k8sutil/k8sutil.go

```diff
@@ -60,6 +78,9 @@ func GetOperatorNamespace() (string, error) {
 
 // GetOperatorName return the operator name
 func GetOperatorName() (string, error) {
+	if isRunModeLocal() {
```

Comment:
Why do we return empty string and error if its local? Maybe someone has set the env variable?

Reply:
Would it be better to move this into the `if !found` block in that case? Also, would it be necessary to put it in the `len(operatorName) == 0` if block as well if we do that or not?

Reply:
Not sure we need this check in the first place. If the env variable is not found, we just error out and error msg is clear enough and user can set the env variable locally as well. Up to you!

Reply:
Removed it and seemed to work fine in local tests. Will retest again in openshift-ci before merging just to make sure.